roseau/load_flow/models/loads/flexible_parameters.py

```diff
@@ -1110,7 +1140,7 @@ def plot_pq(
         power: complex | Q_[complex],
         ax: "Axes | None" = None,
         voltages_labels_mask: NDArray[np.bool_] | None = None,
-    ) -> tuple["Axes", ComplexArray]:
+    ) -> tuple["Axes", ComplexArrayLike1D]:
```

Comment:
Usually "array-like" is for inputs and concrete numpy arrays are used for outputs because we are sure the returned type is ndarray